shared/actions/chat/thread-content.js

```diff
@@ -703,20 +721,6 @@ function* _updateThread({
       yourDeviceName,
       conversationIDKey
     )
-    const messageFromYou =
```

Comment:
This is the code that I just dropped. Please let me know if there is a case it was supposed to handle.

Reply:
Looks like this is from 017a0f5d7006f71e943a7dfcdf3b545e4aa11633 by @cjb. Chris, can you take a look?

Reply:
Ah I remember this bug, let me look at this again.

Reply:
I think there is a bug where we need to find a message with the same outbox ID that has been sent, otherwise it just finds itself all the time and always removes it.

Reply:
I'll put in a better fix than just deleting all this code.

Reply:
Just going to add this line to the `if`:

`message.state !== 'pending'`

Look ok @cjb?